upgrade_analysis/readme/ROADMAP.rst

```diff
@@ -0,0 +1,3 @@
+* scripts/compare_noupdate_xml_records.py should be integrated in the analysis process (#590)
```

Comment:
Oh, this one can be removed. Ready for merge then?

Reply:
> Oh, this one can be removed

Done.